packages/web-pkg/src/components/sideBar/Spaces/SpaceInfo.vue

```diff
@@ -20,14 +20,22 @@
 import { defineComponent } from 'vue'
 
 import { mapGetters } from 'vuex'
+import { PropType } from 'vue'
+import { SpaceResource } from 'web-client'
 
 export default defineComponent({
   name: 'SpaceInfo',
+  props: {
+    spaceResource: {
```

Comment:
Don't we call the prop just `space` usually?

Reply:
It's kinda all over the place. A lot of these space sidebar components have a computed prop `space`, so we do this to avoid name clashing.

Reply:
Wouldn't it make more sense to have a consistent `space` var name in public component API and whatever other var name for internal use?

Reply:
Yeah, at least we keep the API consistent this way. But I'd do this in a follow-up as well, as it probably touches more components.